python/ray/parameter.py

```diff
@@ -92,19 +90,18 @@ class RayParams(object):
     """
 
     def __init__(self,
-                 address_info=None,
```

Comment:
I give up using `address_info` because it is duplicated with other fields like `raylet_socket_name`. Also, a parameter is not a good place to store results. Now one can get `address_info` from `RayNodeSession`